tests/test_numdicts/test_numdicts.py

```diff
@@ -0,0 +1,413 @@
+from itertools import product
+import pyClarion.numdicts as nd
+
+import unittest
+import math
+import itertools
+
+from pyClarion.numdicts.numdicts import NumDict
+
+
+class TestNumdicts(unittest.TestCase):
```

Comment:
`TestNumdicts` seems like too big a test case. It might be better to divide it up into cases like `TestNumDictAddition`, `TestNumDictSubtraction` etc.